pontoon/base/migrations/0047_fix_pluralform_data.py

```diff
@@ -0,0 +1,24 @@
+# Generated manually on 2023-10-10
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("base", "0046_projectslughistory"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+UPDATE base_translation AS tx
+SET plural_form = 0
+FROM base_entity AS ent
+WHERE
+  tx.plural_form IS NULL AND
+  tx.entity_id = ent.id AND
+  ent.string_plural != '';
+"""
```

Comment:
Please rewrite this in Django ORM. We use SQL as the last resort - in case the query is impossible or much more complex to write in Django ORM, or for performance reasons.

Something like this should work:

```python
Translation.objects.filter(plural_form__isnull=True).exclude(entity__string_plural="").update(plural_form=0)
```

Reply:
Can do, but with the caveat that I'm pretty much out of bandwidth atm for Pontoon work given other deadlines. So if I need to rewrite this, that'll _hopefully_ be sometime next week.